lib/colorls/core.rb

```diff
@@ -182,7 +182,7 @@ def mode_info(stat)
 
     def user_info(content)
       user = content.owner.ljust(@userlength, ' ')
-      user.colorize(@colors[:user]) if content.owned?
+      content.owned? ? user.colorize(@colors[:user]) : user
```

Comment:
Nice catch! Is it required to have different colors for users? Or would just `user.colorize(@colors[:user])` do the job?

Reply:
Maybe. It has been this way since its introduction in commit 46f86a4.

@GladOSkar, you wrote this piece of code back in July last year. Do you still use colorls and do you have a strong opinion about this, or why did you do it this way?

Reply:
The idea was that you can quickly see if you own a file without having to read the username. But since i don't use colorls anymore, do whatever you think is best.

Cheers!